lib/lambda_ethereum_consensus/p2p/metadata.ex

```diff
@@ -0,0 +1,55 @@
+defmodule LambdaEthereumConsensus.P2P.Metadata do
```

Comment:
Where is this genserver being started? I think that we should add it to `application.ex`. We also need to add either a `start_link` or `child_spec` public api function for that.